apps/events/models.py

```diff
@@ -11,22 +11,16 @@ def event_thumbmail_path(instance, filename):
 
 class Event(models.Model):
     title = models.CharField(max_length=100)
-    description = models.TextField(max_length=2500)
     schedule = models.DateTimeField()
-    venue = models.CharField(max_length=50)
-    thumbnail = models.ImageField(
+    venue = models.CharField(max_length=50)    
+    contact = models.CharField(max_length=13, blank=True)
+    description = models.TextField(max_length=2500)
+    thumbnail = models.FileField(
         upload_to=event_thumbmail_path, null=True, blank=True)
 
     def __str__(self):
         return self.title
 
-    @property
-    def get_thumbnail_url(self):
-        if self.thumbnail and hasattr(self.thumbnail, 'url'):
-            return self.thumbnail.url
-        else:
-            return settings.STATIC_URL + 'home/images/02.jpg'
-
```

Comment:
Please don't change anything in the models until required. I asked you to refer to the models while creating fields, not to add/remove any fields in models. We don't need a contact field for events. Also, the thumbnail should be an image field, not a file field.